src/__mocks__/nanoid.ts

```diff
@@ -0,0 +1,10 @@
+let id = 0;
+
+beforeEach(() => {
+  id = 0;
+});
+
+export const nanoid = (): string => {
+  id++;
+  return `mock-nanoid-${id}`;
```

Comment:
Added a unique id (in the same way as maas-ui) so that React Testing Library can correctly retrieve the right elements.